specs/DataStructures/Account.md

```diff
@@ -86,6 +89,29 @@ me@google.com    // @ is not allowed (it was allowed in the past)
 abcdefghijklmnopqrstuvwxyz.abcdefghijklmnopqrstuvwxyz.abcdefghijklmnopqrstuvwxyz
 ```
 
+
+## Implicit account IDs
+
+Implicit accounts work similarly to Bitcoin/Ethereum accounts.
+It allows you to reserve an account ID before it's created by generating a ED25519 key-pair locally.
+This key-pair has a public key that maps to the account ID. The account ID is a lowercase hex representation of the public key.
+ED25519 Public key contains 32 bytes that maps to 64 characters account ID.
```

Comment:
`contains` seems to suggest that the key is longer than 32 bytes. Maybe we should say it is 32 bytes instead.